src/accelerate/state.py

```diff
@@ -113,6 +113,19 @@ class PartialState:
     control. Designed to be used when only process control and device execution states are needed. Does *not* need to
     be initialized from `Accelerator`.
 
+    Args:
+        cpu (`bool`, *optional*):
+            Whether or not to force the script to execute on CPU. Will ignore GPU available if set to `True` and force
```

Comment:
> Will ignore GPU available if set to `True` and

Sounds strange, maybe delete  "available"?

Reply:
No, it is a direct force override

Reply:
I'll reword it a bit

Reply:
Let me know how you think it is now